docs/architecture/cloud-native/relational-vs-nosql-data.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Relational vs. NoSQL data
+description: Learn about relational and NoSQL data in cloud-native applications
+author: robvet
+ms.date: 12/30/2019
+---
+# Relational vs. NoSQL data
```

Comment:
These review comments apply to the entire page:
 * Some of the images referenced on this page have too many texts in it. Wherever possible, can you replace them with tabular text instead of an image? Consider accessibility. This will help screen readers to read them.
* Use consistent color and font in the image
* Too many hyperlinks in paras and repetitive ones found (for e.g. ACID))

Reply:
Refactored images and reduced hyperlinks.